src/OpenTelemetry.Extensions.PersistentStorage/README.md

```diff
@@ -16,18 +16,18 @@ dotnet add package OpenTelemetry.Extensions.PersistentStorage
 
 ## Basic Usage
 
-### Setup Storage
+### Setup FileBlobProvider
 
 ```csharp
-using var storage = new FileStorage("test");
+using var fileBlobProvider = new FileBlobProvider("test");
```

Comment:
Do we want to call it `fileBlobProvider` or just `blobProvider`? I feel the common practice is to have the abstraction (e.g. if folks changed this line to a `FoobarBlobProvider` instance, all the subsequent code/logic should continue to work).

Reply:
Abstraction would be persistentBlobProvider - updated it.